installer/templates/phx_assets/webpack.config.js

```diff
@@ -45,9 +36,19 @@ module.exports = (env, options) => {
       ]
     },
     plugins: [
-      new MiniCssExtractPlugin({ filename: '../css/[name].css' }),
-      new CopyWebpackPlugin([{ from: 'static/', to: '../' }])
-    ]
-    .concat(devMode ? [new HardSourceWebpackPlugin()] : [])
+      new MiniCssExtractPlugin({ filename: '../css/app.css' }),
+      new CopyWebpackPlugin({
+        patterns: [
+          { from: 'static/', to: '../' }
+        ]
+      })
+    ],
+    optimization: {
+      minimizer: [
+        '...',
+        new CssMinimizerPlugin()
+      ]
+    },
```

Comment:
I also think you will get compilation errors if you don't specify `target: 'nodeX.Y',` because the default option for `target` is `web` which will usually cause things like `http` and `url` to blow up upon upgrading. Since these assets are built in a node-like environment targeting Node over web will ensure Webpack successfully compiles.

Reply:
That's a thing I for myself would need to explore more in detail. I didn't get the consequences of this now.
For Phoenix deploys the `target` in my opinion should be `web` - not `node`.
But you might be right.

Reply:
I thought the same too but when I deployed with `web` I was getting errors around `url` and `http` and you'd have to set resolutions that resolve to `false` for both. Since this is a `node` environment for how we build our assets if you change it to `node` everything will resolve and pass.